### PR TITLE
IDEM-501

### PR DESCRIPTION
**Pull request by xiangzhe0812**
individual views for the claims

### CODE REVIEW

File: app/src/navigation/ProfileStackNavigator.tsx [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
-import ClaimScreen from "./screens/ClaimScreen";
-import { ClaimType } from "../types/claim";
-import { getClaimFromType } from "../utils/claim-utils";
 import ViewFile from "./screens/ViewFileScreen";
+import NameClaimScreen from "./screens/ClaimScreens/NameClaimScreen";
+import BirthClaimScreen from "./screens/ClaimScreens/BirthClaimScreen";
+import MobileClaimScreen from "./screens/ClaimScreens/MobileClaimScreen";
+import AddressClaimScreen from "./screens/ClaimScreens/AddressClaimScreen";
+import AdultClaimScreen from "./screens/ClaimScreens/AdultClaimScreen";
+import EmailClaimScreen from "./screens/ClaimScreens/EmailClaimScreen";
```
Comment (bitcoinbrisbane) [inline app/src/navigation/ProfileStackNavigator.tsx:10]: May aswell do some gardening...

```suggestion
import AddressClaimScreen from "./screens/ClaimScreens/AddressClaimScreen";
import AdultClaimScreen from "./screens/ClaimScreens/AdultClaimScreen";
import BirthClaimScreen from "./screens/ClaimScreens/BirthClaimScreen";
import EmailClaimScreen from "./screens/ClaimScreens/EmailClaimScreen";
import MobileClaimScreen from "./screens/ClaimScreens/MobileClaimScreen";
import NameClaimScreen from "./screens/ClaimScreens/NameClaimScreen";
```

File: app/src/navigation/screens/DocumentTypesScreen.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ import { useNavigation } from "@react-navigation/native";
 import { DocumentsStackNavigation } from "../../types/navigation";
 import { useDocumentStore } from "../../context/DocumentStore";
 
-type Navigation = DocumentsStackNavigation<"Document Types">;
+type Navigation = DocumentsStackNavigation<"Documents">;
```
Comment (bitcoinbrisbane) [inline app/src/navigation/screens/DocumentTypesScreen.tsx:12]: I think this should be document types
```suggestion
type Navigation = DocumentsStackNavigation<"Document Types">;
```

File: app/src/utils/claim-utils.ts [hunk 3]
```diff
@@ -9,6 +11,36 @@ import {
...
+    case "MobileCredential":
+      return "MobileClaim";
+    case "BirthCredential":
+      return "BirthClaim";
+    case "AddressCredential":
+      return "AddressClaim";
+    case "AdultCredential":
+      return "AdultClaim";
+    case "EmailCredential":
+      return "EmailClaim";
```
Comment (bitcoinbrisbane) [inline app/src/utils/claim-utils.ts:27]:
```suggestion
    case "AddressCredential":
      return "AddressClaim";
    case "AdultCredential":
      return "AdultClaim";
    case "BirthCredential":
      return "BirthClaim";
    case "EmailCredential":
      return "EmailClaim";
   case "MobileCredential":
      return "MobileClaim";
    case "NameCredential":
      return "NameClaim";
```

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
Few small things

**Review by amagon270 — DISMISSED** [review]

**Review by amagon270 — APPROVED** [review]

**Pull request merged**